linode_api4/objects/account.py

```diff
@@ -306,6 +394,7 @@ def get_obj_grants():
         ("volume", Volume),
         ("image", Image),
         ("longview", LongviewClient),
+        ("database", Database),
```

Comment:
I did a bit of digging and it looks like `firewall` is also a supported grant type, although it's not documented in the API spec.